server/bin/server.py

```diff
@@ -21,10 +21,13 @@
 import redis
 import tornado.ioloop
 from tornado.options import define, options
+from dotenv import load_dotenv
```

Comment:
Keep the alphabetical order in the imports list.

Reply:
It was placed like this to avoid pylint warning ' C0412: Imports from package tornado are not grouped (ungrouped-imports)'.